angular/core/components/group_page/group_previous_proposals_card/group_previous_proposals_card.coffee

```diff
@@ -16,3 +16,21 @@ angular.module('loomioApp').directive 'groupPreviousProposalsCard', ->
 
     $scope.canShowMore = ->
       $scope.group.closedMotionsCount > 3
+
+angular.module('loomioApp').filter 'cut', ->
```

Comment:
@reaksmey1 Any update on this? Would be great to merge this in, with that minor change!